primo/utils/kpi_utils.py

```diff
@@ -22,6 +22,15 @@
 from primo.utils.raise_exception import raise_exception
 
 
+def _check_column_name(data: pd.DataFrame, col_name: str):
```

Comment:
To ensure it happened :), you're right, I'll take it out. Although, on this point, why then do we ever check that a column exists in a dataframe? Should we always default to the pandas error? (Maybe it is just me that is checking column names unnecessarily...) There's another function that I added to do this. I'll remove it as well.

Reply:
This definitely simplifies things and makes the code cleaner. I vote we just default to pandas for checking everything. The only potential downside I see is that we could give more specific information about what is missing and what to do about it (although I suppose we could always catch the error and provide more information if we really wanted to)

Reply:
This would also apply to the above conversation surrounding the "num_wells_near_hospitals" attribute. I am going to go on the premise that simpler is better and simply get rid of the checks for column names and let pandas do the checking.

Reply:
Discussed over slack.